posthog/tasks/update_cache.py

```diff
@@ -117,9 +117,7 @@ def task_for_cache_update_candidate(candidate: Union[DashboardTile, Insight]) ->
 
 def gauge_cache_update_candidates(dashboard_tiles: QuerySet, shared_insights: QuerySet) -> None:
     statsd.gauge("update_cache_queue.never_refreshed", dashboard_tiles.filter(last_refresh=None).count())
-    oldest_previously_refreshed_tiles: List[DashboardTile] = list(
-        dashboard_tiles.exclude(last_refresh=None)[0:PARALLEL_INSIGHT_CACHE]
-    )
+    oldest_previously_refreshed_tiles: List[DashboardTile] = list(dashboard_tiles.exclude(last_refresh=None)[0:10])
```

Comment:
Is this meant to be hardcoded to 10?

Reply:
Yep... to save reading the instance setting twice. 

We don't care how many we gauge just "more than one" and "less than thousands" 🤣